apps/explorer/src/pages/epochs/useEpochProgress.ts

```diff
@@ -6,18 +6,14 @@ import { useQuery } from '@tanstack/react-query';
 
 export function useEpochProgress(suffix: string = 'left') {
     const rpc = useRpcClient();
-    const { data } = useQuery(
-        ['system', 'state'],
-        () => rpc.getLatestSuiSystemState(),
-        {
-            refetchInterval: 5000,
-        }
+    const { data } = useQuery(['system', 'state'], () =>
+        rpc.getLatestSuiSystemState()
     );
 
     const start = data?.epochStartTimestampMs ?? 0;
     const duration = data?.epochDurationMs ?? 0;
     const end = start + duration;
-    const time = useTimeAgo(end, true);
+    const time = useTimeAgo(end, true, true);
```

Comment:
You can pass the end label to useTimeAgo. `useTimeAgo(end, true, true, 'Ending soon')`.

Reply:
we use a suffix here so we need to do the check regardless in the `label` prop, otherwise we end up with `Ending soon ${suffix}`